Content.Server/Nyanotrasen/StationEvents/Events/GlimmerWispSpawnRule.cs

```diff
@@ -25,7 +27,7 @@ protected override void Started(EntityUid uid, GlimmerWispRuleComponent componen
         var hiddenSpawnLocations = EntityManager.EntityQuery<MidRoundAntagSpawnLocationComponent, TransformComponent>().ToList();
 
```

Comment:
Could probably rewrite some of this with the suggestions I made in the other comment.